test/units/modules/cloud/amazon/placebo_fixtures.py

```diff
@@ -0,0 +1,192 @@
+from nose.plugins.skip import SkipTest
```

Comment:
New tests should use `pytest` features since we're migrating away from `nose`. Take a look at the following alternative to `SkipTest`:

http://pytest.org/dev/skipping.html#skipping-on-a-missing-import-dependency